responses.py

```diff
@@ -0,0 +1,14 @@
+class GoodMorning:
+    def process(self, message,client):
+        if message.author == client.user:
```

Comment:
Are we sure that it is going to stop the loop?

Reply:
Should do the work. There is no messaging anyways so i am removing it for now it is not needed.